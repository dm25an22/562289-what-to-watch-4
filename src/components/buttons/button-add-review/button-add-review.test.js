import React from "react";
import renderer from "react-test-renderer";
import ButtonAddReview from "./button-add-review.jsx";


it(`render ButtonAddReview component`, () => {
  const tree = renderer.create(
      <ButtonAddReview />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
