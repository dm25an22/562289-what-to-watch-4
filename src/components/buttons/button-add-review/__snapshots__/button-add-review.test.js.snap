// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ButtonAddReview component 1`] = `
<a
  className="btn movie-card__button"
  href="/add-review/3"
  onClick={[Function]}
>
  Add review
</a>
`;
